<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_identificationmanufacturer</class>
 <widget class="QDialog" name="dlg_identificationmanufacturer">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>626</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Identification fabricant</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="Principalframe">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>430</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>430</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="UpLineEdit" name="FaxlineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>336</y>
      <width>170</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="UpLineEdit" name="TellineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>246</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>17</number>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>226</y>
      <width>475</width>
      <height>3</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="UpLineEdit" name="NomlineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>8</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="maxLength">
     <number>45</number>
    </property>
   </widget>
   <widget class="QLabel" name="Faxlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>145</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fax</string>
    </property>
   </widget>
   <widget class="UpLineEdit" name="MaillineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>306</y>
      <width>240</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>45</number>
    </property>
   </widget>
   <widget class="QLabel" name="Adresse2label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>145</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Complément d'adresse</string>
    </property>
   </widget>
   <widget class="QLabel" name="Maillabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>145</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="Adresse3label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>145</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Complément d'adresse</string>
    </property>
   </widget>
   <widget class="UpLineEdit" name="Adresse1lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>66</y>
      <width>340</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>70</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Tellabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>145</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Téléphone</string>
    </property>
   </widget>
   <widget class="QLabel" name="Portlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>145</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Portable</string>
    </property>
   </widget>
   <widget class="UpLineEdit" name="Adresse2lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>96</y>
      <width>340</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>70</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="UpLineEdit" name="Adresse3lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>126</y>
      <width>340</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>70</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Adresse1label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>145</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>44</y>
      <width>475</width>
      <height>3</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Nomlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>145</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="UpLineEdit" name="PortablelineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>276</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>17</number>
    </property>
   </widget>
   <widget class="QLabel" name="Websitelabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>145</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Web</string>
    </property>
   </widget>
   <widget class="UpLineEdit" name="WebsiteineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>366</y>
      <width>240</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>45</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="ActifcheckBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Activer/Desactiver ce fabricant</string>
    </property>
    <property name="text">
     <string>Actif</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Distributeurlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>145</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Distribué par</string>
    </property>
   </widget>
   <widget class="UpComboBox" name="DistributeurupComboBox">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>392</y>
      <width>250</width>
      <height>32</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="idManufacturerlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dossier n°</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="UpTableView" name="commercialsupTableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>550</width>
     <height>140</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UpLineEdit</class>
   <extends>QLineEdit</extends>
   <header>uplineedit.h</header>
  </customwidget>
  <customwidget>
   <class>UpComboBox</class>
   <extends>QComboBox</extends>
   <header>upcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>UpTableView</class>
   <extends>QTableView</extends>
   <header>uptableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>NomlineEdit</tabstop>
  <tabstop>Adresse1lineEdit</tabstop>
  <tabstop>Adresse2lineEdit</tabstop>
  <tabstop>Adresse3lineEdit</tabstop>
  <tabstop>TellineEdit</tabstop>
  <tabstop>PortablelineEdit</tabstop>
  <tabstop>MaillineEdit</tabstop>
  <tabstop>FaxlineEdit</tabstop>
  <tabstop>WebsiteineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
